templates/index.html

```diff
@@ -25,173 +25,172 @@ <h1> OpenStack for the edge, <br/> micro clouds and developers </h1>
         <iframe class="u-embedded-media__element" width="560" height="315" src="https://www.youtube.com/embed/fNk7AfxNggc" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
       </div>
   </section>
+
   <section class="p-strip">
     <div class="row">
       <div class="p-notification--caution">
         <div class="p-notification__content">
           <p class="p-notification__message">
-            <strong>Note:</strong> MicroStack is in a beta state. We encourage you to test it, give us your <a href="https://bugs.launchpad.net/microstack">feedback</a> and <a href="https://discourse.juju.is/search?q=microstack">ask questions</a>. For information about commercial support for MicroStack, <a href="https://ubuntu.com/openstack/contact-us">contact Canonical</a>.
+            <strong>Note:</strong> MicroStack is in a beta state. We encourage you to test it, share your <a href="https://microstack.run/survey">feedback</a> and <a href="https://discourse.juju.is/search?q=microstack">ask questions</a>. For commercial support, <a href="/openstack/contact-us">contact us directly</a>.
```

Comment:
Could you use the [Information Notification](https://vanillaframework.io/docs/patterns/notification#information) pattern here please?